### PR TITLE
Fixed choice_translation_domain for expanded choices

### PR DESCRIPTION
**Pull request by Hanmac**
I am targeting this branch, because it fixed the translation domain for expanded choices in admin.

imo that was a bug which is fixed by this backwards compatible thing

## Changelog

```markdown

### Fixed
- fixed choice_translation_domain for expanded choices in admin
- make false translation_domain working for the label if no translation is needed
```

### CODE REVIEW

File: Resources/views/Form/form_admin_fields.html.twig [hunk 1]
```diff
@@ -183,7 +187,7 @@ file that was distributed with this source code.
     <ul {{ block('widget_container_attributes') }}>
     {% for child in form %}
         <li>
-            {{ form_widget(child, {'horizontal': false, 'horizontal_input_wrapper_class': ''}) }} {# {'horizontal': false, 'horizontal_input_wrapper_class': ''} needed to avoid MopaBootstrapBundle messing with the DOM #}
+            {{ form_widget(child, {'horizontal': false, 'horizontal_input_wrapper_class': '', 'translation_domain': choice_translation_domain}) }} {# {'horizontal': false, 'horizontal_input_wrapper_class': ''} needed to avoid MopaBootstrapBundle messing with the DOM #}
```
Comment (greg0ire) [inline Resources/views/Form/form_admin_fields.html.twig]: Please linebreak this

**Comment by greg0ire** [comment]
Please format your commit message according to our rules : 

- The commit message subject must be less than 50 characters and tell us *what* you did.
- The commit message body should tell us *why* you did it. It is optional but highly recommended.

Bad example :

```
Fixed bug #989 by removing call to defraculate()
```

Good example:

```
Remove call to defraculate()

Calling this function caused a bug because it interferes 
with calls to getSchmeckles().
Fixes #989
```

More details in CONTRIBUTING.md

**Comment by Hanmac** [comment]
i think i messed it up with the repository,
i make a new cleaner branch for that, but i don't seems to be able to change the pr anymore

https://github.com/Hanmac/SonataAdminBundle/tree/choice_translation_domain

i think i need to make a new PR for this.

**Comment by greg0ire** [comment]
>  from unknown repository

yeah this looks bad :P

**Comment by core23** [comment]
Duplicate of #4670

**Pull request closed without merging**